src/PKSim.Presentation/DTO/Individuals/IndividualSettingsDTO.cs

```diff
@@ -14,12 +14,18 @@ public class IndividualSettingsDTO : IWithValueOrigin
       public IParameterDTO ParameterHeight { get; private set; }
       public IParameterDTO ParameterBMI { get; private set; }
 
-      public SpeciesPopulation SpeciesPopulation { get; set; }
+      public SpeciesPopulation Population { get; set; }
       public IEnumerable<CategoryParameterValueVersionDTO> SubPopulation { get; set; }
       public IEnumerable<CategoryCalculationMethodDTO> CalculationMethods { get; set; }
       public Species Species { get; set; }
       public Gender Gender { get; set; }
       public ValueOrigin ValueOrigin { get; }
+      public DiseaseState DiseaseState { get; set; }
+      
+      /// <summary>
+      /// For now, we assume only one disease state parameters. We can make the view more generic if this is ever required
+      /// </summary>
+      public IParameterDTO DiseaseStateParameter { get; set; }
```

Comment:
I am assuming for now that we only have ONE parameter as I did not want to spend too much time dealing with multiple entries in the view etc.. if we have a disease state with multiple inputs...then we can update the DTO (keep it simple). The model has multiples parameters. Just the view shows one